scripts/generate_activity_script/src/script.ts

```diff
@@ -0,0 +1,353 @@
+  console.error(`0 <= BOND_PROB(${BOND_PROB}) <= 1 != true`);
+  process.exit(1);
+}
+if ((0 <= UNBOND_PROB && UNBOND_PROB <= 1) === false) {
+  console.error(`0 <= UNBOND_PROB(${UNBOND_PROB}) <= 1 != true`);
+  process.exit(1);
+}
+if ((0 <= WITHDRAW_PROB && WITHDRAW_PROB <= 1) === false) {
+  console.error(`0 <= WITHDRAW_PROB(${WITHDRAW_PROB}) <= 1 != true`);
+  process.exit(1);
+}
+
+enum MODE {
+  BOND = "BOND",
+  UNBOND = "UNBOND",
+  WITHDRAW = "SEND_NFT",
+}
+
+type Wallet = {
+  mainWallet?: DirectSecp256k1HdWallet;
+  clientCW?: SigningCosmWasmClient;
+  clientSG?: SigningStargateClient;
+  mainAccounts?: readonly AccountData[];
+};
+
+type Action = {
+  mode: MODE;
+  txHash: string;
+};
+
+async function bond(
+  drop_instance: DropCoreClient,
+  address: string,
+  fund: Coin
+): Promise<Action> {
+  return {
+    mode: MODE.BOND,
+    txHash: (
+      await drop_instance.bond(address, {}, "auto", "", [
+        {
+          amount: fund.amount,
+          denom: fund.denom,
+        },
+      ])
+    ).transactionHash,
+  };
+}
+
+/* Function returns ActionLog if transaction succeeded and null if any error or problem occurred
+ * We don't really need to know type of error here since it isn't crucial for us,
+ * We'll just try another method in core
+ */
+async function bond_random_amount(
+  neutronWallet: Wallet,
+  drop_instance: DropCoreClient
+): Promise<Action | null> {
+  const address: string = neutronWallet.mainAccounts[0].address;
+
+  /* If here is nothing to bond on our balance, then just return null
+   * Other random method will be tried to call then
+   */
+  let ibc_denom_balance: Coin = await neutronWallet.clientCW.getBalance(
+    address,
+    BASE_DENOM
+  );
+  if (Number(ibc_denom_balance.amount) === 0) {
+    return null;
+  }
+
+  const config = await drop_instance.queryConfig();
+  const exchange_rate: number = Math.floor(
+    Number(await drop_instance.queryExchangeRate())
+  );
+  const min_exchange_rate: number = exchange_rate + 1;
+
+  /* Minimum amount of funds that we can send to core contract while bonding
+   * Is either lsm_min_bond_amount (which is typically set in 1) or current exchange_rate parameter
+   * Here we're choosing the biggest value of these two to avoid further errors
+   */
+  const min: number =
+    Number(config.lsm_min_bond_amount) < min_exchange_rate
+      ? min_exchange_rate
+      : Number(config.lsm_min_bond_amount);
+  if (min > Number(ibc_denom_balance.amount)) {
+    return null;
+  }
+
+  /* If any error occured when executing method then just ignore
+   * It's content and return null, script will try to call another method
+   */
+  let random_amount: number = Math.floor(
+    Math.random() * (Number(ibc_denom_balance.amount) - min) + min
+  );
+  try {
+    const res = await bond(drop_instance, address, {
+      amount: String(random_amount),
+      denom: BASE_DENOM,
+    });
+    if ((await neutronWallet.clientCW.getTx(res.txHash)).code !== 0) {
```

Comment:
```suggestion
    const { code } = await neutronWallet.clientCW.getTx(res.txHash);
    if (code !== 0)
```
I noticed that `(await ...).field` design is often used, but I think that the code in the style above is better readable